joomla/joomla-cms
# [3.10] Add a check whether we are running the core backend template and issue an info when that is not the case

### Summary of Changes

Add a check whether we are running the core tempaltes where the backend it required and the frontend is recommended

### Testing Instructions

- Install 3.10.0
- install a non-core backend template (https://www.joomlatema.net/joomla-templates/free-joomla-templates/adminim-free-joomla-administrator-template.html)
- set the template as default for the backend
- apply this patch
- point the update server to joomla 4 via the Joomla Next update server
- notice the new message

### Actual result BEFORE applying this Pull Request

No checks for the default templates

![image](https://user-images.githubusercontent.com/2596554/129271517-e7fdb9a3-7a12-4c2f-9e05-041f03efbb65.png)


### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/2596554/129329145-522cac3a-4389-4883-b753-9aa3c220f9b1.png)


### Documentation Changes Required

https://docs.joomla.org/Pre-Update_Check - Has to be updated

TTs have to be informed about the language stirngs

cc @joomla/cms-release-team @HLeithner @bembelimen
